docs/doc/30-reference/10-data-types/60-data-type-nullable-types.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Nullable
+description: Nullable data type.
+---
+
+## Nullable Data Types
+
+`NULL` values are used to represent nonexistent or unknown values.
+`Nullable` data types allow storing `NULL` values.
+For example, the `Nullable(Int64)` type can store `Int64` values as well as `NULL`.
+
+## Syntax
+
+Define column field data types `Nullable` while creating a table is as follows:
+
+```sql
+CREATE TABLE test(
+    id Int64,
+    name String NULL,
+    age Int32 NOT NULL
+);
+```
+
+`NULL` indicates that the data type of this field is `Nullable` and can store `NULL` values.
+`NOT NULL` indicates that this field cannot store `NULL` values.
+If not explicitly set to be `Nullable`, it defaults to `NOT NULL`.
+
+:::note
+Using `Nullable` will almost always have a negative impact on performance. Setting it to `NOT NULL` is a better option.
```

Comment:
The user also wants to know: what's the Default behavior for the databend column?